docs/ansible_inventory.md

```diff
@@ -0,0 +1,103 @@
+# Ansible Inventory Files
+
+## Brent Resource Package
+
+The "config" subdirectory in the Brent resource package driver root directory contains Ansible inventory and ancillary configuration files. The Ansible Lifecycle Driver expects an inventory file named "inventory" in this directory (for SSH connections using the [Ansible SSH connection plugin](https://docs.ansible.com/ansible/latest/plugins/connection.html#ssh-plugins)) and/or a file named "inventory.k8s" (for K8s connections using the [Ansible kubectl connection plugin](https://docs.ansible.com/ansible/latest/plugins/connection/kubectl.html)).
+
+The driver determines which inventory file to use based on the deployment location type (i.e. "deploymentLocation.type") in the lifecycle request. If the type is "Kubernetes" the driver will expect an "inventory.k8s" file. If the type is anything other than "Kubernetes", an SSh connection and "inventory" file are expected.
+
+Note: if you wish to split your inventory out in to separate host variable files then you may do so. For example:
+
+```
+config
+  host_vars
+    host1.yml
+  inventory
+  inventory.k8s
+```
+
+## Variable Substitution
+
+The Ansible Lifecycle Driver supports the substitution of LM properties in inventory files under the "config" directory, using [Jinja2 template variables](https://jinja.palletsprojects.com/en/2.10.x/templates/#variables). The following properties are available:
+
+* properties: the set of LM request properties.
+* system_properties: the set of system properties.
+* dl_properties: deployment location properties.
+
+For example, the following inventory file substitutes the LM property "properties.mgmt_ip_address" for "ansible_host":
+
+```
+---
+ansible_user: ubuntu
+ansible_host: {{ properties.mgmt_ip_address }}
```

Comment:
I guess all of these need updating to "{{{" with PR #24 ?

Reply:
PR #24 now closed, not needed